### PR TITLE
nix improvements

### PR DESCRIPTION
**Pull request by disassembler**

### CODE REVIEW

File: lib.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    let try = builtins.tryEval <iohk_nix>;
+    in if try.success
+    then builtins.trace "using host <iohk_nix>" try.value
+    else
+      let
+        spec = builtins.fromJSON (builtins.readFile ./iohk-nix.json);
+      in builtins.fetchTarball {
+        url = "${spec.url}/archive/${spec.rev}.tar.gz";
+        inherit (spec) sha256;
+      }) {};
```
Comment (jmitchell) [inline lib.nix:14]: Very nifty interface for pulling in iohk-nix. 👍

**Review by jmitchell — DISMISSED** [review]

**Comment by disassembler** [comment]
bors ping

**Comment by disassembler** [comment]
bors try

**Comment by disassembler** [comment]
bors try

**Comment by disassembler** [comment]
bors try

**Comment by disassembler** [comment]
bors try

**Review by cleverca22 — APPROVED** [review]

**Pull request merged**